wolfSSL/src/chapter02.md

```diff
@@ -472,17 +604,53 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
-#### WOLFSSL_RIPEMD
+#### WOLF_PRIVATE_KEY_ID
+
+This is used with PKCS11 to enable support for key ID and label API's. FIPS v5 and older doesn't support `WOLF_PRIVATE_KEY_ID` with PK callbacks.
+
+#### WOLFSSL_WOLFSENTRY_HOOKS
+
+Enables wolfSentry hook support.
```

Comment:
@douzzer anything you want to add?

Reply:
This switch adds support in the TLS layer for generic network `accept` and `connect` filter hooks, using `wolfSSL_CTX_set_AcceptFilter()` and `wolfSSL_CTX_set_ConnectFilter()`.  It also activates wolfSentry integration in the example client and server applications.